feature-libs/organization/administration/core/services/org-unit.service.ts

```diff
@@ -249,10 +249,19 @@ export class OrgUnitService {
 
   getActiveUnitList(): Observable<B2BUnitNode[]> {
     return this.getList().pipe(
-      map((units) => units.filter((unit) => unit.active))
+      map((units) => units.filter((unit) => unit.active)),
+      map((units) => units.sort(this.sortUnitList))
     );
   }
 
+  sortUnitList(a: B2BUnitNode, b: B2BUnitNode) {
```

Comment:
Need a unit test for this.

Reply:
yup + IMO it can be `protected` method, but sorting can be verified in test for `getActiveUnitList`

Reply:
Got it. I will add the changes.